issues.md

```diff
@@ -0,0 +1,38 @@
+Reporting issues
+================
+
+By default, all Issues logged in Github are meant to be considered to be a _bug_, meaning, _a software problem affecting the stability of the product_. This includes functionalities that run without error, but do something unexpected which also works against the system's stability.
+
+Priority labels
+---------------
+In September 2016, Ministry Brands helped define priority levels for issues based on an overall Service Level Agreement (SLA). We use the labels P1, P2, P3 and P4 define the overall _priority_ of resolution on the issue.
+
+**P1** - Urgent. A key system component or feature is broken. These are the team's highest priority until resolved.
+
+**P2** - Important. Should be fixed in the next 2-4 weeks.
+
+**P3** - Fix eventually (within 2-3 months).
+
+**P4** - A nice-to-have; will add as time allows. Mostly includes enhancements that aren't often requested, but would drive value in the product especially if released along with other P4's.
+
+Other labels
+------------
+All other labels are meant to take an Issue beyond the context of a _bug_.
+
+**enhancement** - An iterative improvement to a module which is already functioning normally.
+
+**support** - Issues which are tied to client feedback which would benefit from a personal response.
+
+**security** - Issues that could have an effect on the privacy of client information.
+
+_Note: Adding any of these labels doesn't change the apparent priority of an issue. It is possible to have a high-priority "enhancement"._
+
+Feature requests
+----------------
+Issues should not be used as an inbox for client feature requests. Instead, each member of the team should be comfortable acting as a kind of gatekeeper for the requests that come in from the clientbase, working across the teams to determine the potential value of the feature.
+
+Requests for entirely new features should be fleshed out in a separate document, outside of Issues. The _Voice of the Customer_ meetings are the best place to flesh out your ideas with the general managers.
```

Comment:
I think this paragraph goes beyond what needs to be known about issues. If the process were to change for feature requests, we'd likely forget to update the text here.